app/views/protocols/index/_users_list.html.erb

```diff
@@ -0,0 +1,12 @@
+<% protocol.user_assignments[0..3].each do |user_assigment| %>
```

Comment:
I think you need to rebase the branch because that code already exists in feature branch.

I modified it in my PR - https://github.com/scinote-eln/scinote-web/pull/4853/files